cmd/manager/main.go

```diff
@@ -214,7 +217,12 @@ func newRootCommand() *cobra.Command {
 				log.Info("Starting the Cmd.")
 
 				// Start the Cmd
-				log.Fatal(mgr.Start(signals.SetupSignalHandler()))
+				err = mgr.Start(signals.SetupSignalHandler())
+				if err != nil {
+					log.Fatal(err)
```

Comment:
The leader election context should be cancelled whenever `mgr.Start` returns, regardless of whether it returned an error or not.

Reply:
replacing `log.Fatal` with `log.Error` here